doc/source/starting-ray.rst

```diff
@@ -8,38 +8,80 @@ This page covers how to start Ray on your single machine or cluster of machines.
 Installation
 ------------
 
-Install Ray with ``pip install -U ray``. For the latest wheels (a snapshot of the ``master`` branch), you can use the instructions at :ref:`install-nightlies`.
+.. tabs::
+  .. group-tab:: Python
+
+    Install Ray with ``pip install -U ray``. For the latest wheels (a snapshot of the ``master`` branch), you can use the instructions at :ref:`install-nightlies`.
+
+  .. group-tab:: Java
+
+    Add `Ray API <https://mvnrepository.com/artifact/io.ray/ray-api>`_ and `Ray Runtime <https://mvnrepository.com/artifact/io.ray/ray-runtime>`_ as dependencies. We don't publish snapshot versions right now. Note that to start a multi-node Ray cluster, you'll also need to follow the installation instructions of Python version.
```

Comment:
We should explain that `pip install` will also install Java jars. And these dependencies are used to compile code and run Ray on a single machine.